src/hotspot/share/oops/constantPool.hpp

```diff
@@ -485,26 +483,6 @@ class ConstantPool : public Metadata {
   // Version that can be used before string oop array is created.
   oop uncached_string_at(int which, TRAPS);
 
-  // A "pseudo-string" is an non-string oop that has found its way into
-  // a String entry.
-  // This can happen if the user patches a live
-  // object into a CONSTANT_String entry of an unsafe anonymous class.
-  // Methods internally created for method handles may also
-  // use pseudo-strings to link themselves to related metaobjects.
```

Comment:
Is this comment wrong? Are psuedo-strings not used by anything now?

Reply:
Thanks for looking at this.  I discussed pseudo strings with Coleen and we didn't find any use of them besides unsafe.DefineAnonymousClass.